### ISSUE TITLE
Error with Ogg vorbis 

### ISSUE THREAD

**Issue by beudbeud** [libretro/scummvm#58]
Hi, i have a issue with libretro scummvm core, 
I get message "Music for thi sgame requires Ogg Vorbis support". I went to scummvm discord and they told me to come here for my issue. 

I compile libretro-scummvm on linux with builroot, i used default options is not work

I tried too with many options like USE_SYSTEM_vorbis=1 USE_OGG=1 USE_VORBIS=1 but nothing change.

```
UILD_64BIT=1 USE_MT32EMU=1 USE_SYSTEM_vorbis=1 USE_OGG=1 USE_VORBIS=1
- Use system shared vorbis: yes
Compiling vorbis.cpp...
```

**Comment by spleen1981** [comment]
Can you give more detail, like platform, engine where this occurs, etc?
Also please provide full build log and retroarch.log with debug level set both for frontend and core.

**Comment by beudbeud** [comment]
Platform unix build (rpi4 64bits) with buildroot, i get this error on "[Schizm: Mysterious Journey](https://www.scummvm.org/compatibility/DEV/vcruise:schizm/)"

[build.log](https://github.com/user-attachments/files/16024984/build.log)
[retro.log](https://github.com/user-attachments/files/16025058/retro.log)

**Comment by beudbeud** [comment]
i tried another game with ogg like Gobliiins and monkey island and is work well, only for Schizm: Mysterious Journey, i get this error message

**Comment by spleen1981** [comment]
fixed by 4f5f776b8d25cfc4636ff02e01b8fa7fe4259950
